### ISSUE TITLE
Bug: Endless spin after finishing onboarding step 3

### ISSUE THREAD

**Issue by cwonrails** [TryGhost/Ghost#5779]
### Issue Summary

If the user has not already set up their email in config.js, inviting multiple team members during onboarding fails and the user is _not_ directed to the admin panel. All they see is this endless spin:

![post-onboarding endless spin](https://cloud.githubusercontent.com/assets/4729979/9608402/cb083fa4-5093-11e5-98a1-a86f810c63d1.gif)
### Steps to Reproduce
1. `git clone git@github.com:TryGhost/ghost.git`
2. `cd ghost && npm install && grunt init && npm start`
3. Go through onboarding process, avoiding all errors.
4. Enter multiple unique emails in step 3 of onboarding process.

This is a bug because:

User should be taken directly to admin panel post-failure so they can follow the prompts for setting up email correctly. Following the directions for [Developer Install (from git)](https://github.com/TryGhost/Ghost#developer-install-from-git) from the README will lead to this error.
### Technical details
- Ghost Version: master - latest commit:  40f82b6
- Client OS: Mac OS X 10.11
- Server OS: same (localhost generated by npm start)
- Node Version: v0.12.7
- npm Version: 2.11.3
- Browser: Chrome Version 44.0.2403.157 (64-bit)
- Database: SQLite 3

**Comment by ErisDS** [comment]
To improve the UX here we need to better handle the way emails are sent. We don't need to wait for them to be sent before transitioning, however that may mean the notifications are not shown at the right time. 

We may need to revisit how and when notifications are pulled from the server, and how they are handled in order to resolve this fully?

**Comment by vdemedes** [comment]
There is an endless spin, because requests are not closed, even after the failed attempt to send an email. It took ~77 seconds for each request to close. So there is an issue with error handling, because `ETIMEDOUT` error causes the request to close.

Perhaps we should check if email is configured before even trying to send?

**Comment by vdemedes** [comment]
I think the immediate transition after clicking "Invite" button to admin panel is a preferable default, since we will see notifications for failed emails anyway:

<img width="471" alt="screen shot 2015-10-20 at 4 29 42 pm" src="https://cloud.githubusercontent.com/assets/697676/10610381/d4d213c2-7747-11e5-858b-90ceffd9d3ea.png">

**Comment by cwonrails** [comment]
+1 for @vdemedes comment

**Comment by vdemedes** [comment]
In my opinion, these 2 improvements combined could solve this:

1. Introduce a 4-6 second timeout to `send()` mail method
2. Immediately transition to dashboard after invitation step

**Comment by srrvnn** [comment]
this seems to have been fixed. i finished the setup with multiple emails in step three and was immediately transitioned to the dashboard page. 

also, in an attempt to reset so that i can setup again when i go to /ghost, i ran `grunt cleanDatabase`. that fails with :

```
Fatal error: TypeError: Cannot read property 'schema' of undefined
    at Object.deleteTable (/Users/saravananganesh/Projects/ghost/core/server/data/utils/index.js:78:25)
```

how can i setup `dbConfig.knex`, and what am i missing?

**Comment by ErisDS** [comment]
Closing as this was fixed in #6307
